ts-rs/Cargo.toml

```diff
@@ -54,10 +56,10 @@ bson = { version = "2", optional = true }
 bytes = { version = "1", optional = true }
 url = { version = "2", optional = true }
 semver = { version = "1", optional = true }
-smol_str = { version = "0.3", optional = true  }
+smol_str = { version = "0.3", optional = true }
 thiserror = "1"
 indexmap = { version = "2", optional = true }
 ordered-float = { version = ">= 3, < 5", optional = true }
 serde_json = { version = "1", optional = true }
 lazy_static = { version = "1", default-features = false }
-
+tokio = { version = "1.40", optional = true }
```

Comment:
Can we support an older version of tokio? I think even 1.0 should be fine?

Reply:
Yes, I updated dependency.